packages/common-common/src/rabbitmq/types/snapshotNotification.ts

```diff
@@ -0,0 +1,35 @@
+import {RmqMsgFormatError, RmqMsgNamespace} from "common-common/src/rabbitmq";
+import {SnapshotNotification} from "../../../../commonwealth/shared/types"
```

Comment:
common-common shouldn't have any dependencies on the commonwealth package.

Reply:
for rich

Reply:
added the type into common-common, no more circular import